docs/new-architecture-app-modules-android.md

```diff
@@ -212,10 +266,47 @@ public class MyApplication extends Application implements ReactApplication {
 }
 ```
 
+</TabItem>
+<TabItem value="kotlin">
+
+```kotlin
+class MyApplication : Application(), ReactApplication {
+    private val mReactNativeHost: ReactNativeHost =
+        object : ReactNativeHost(this) {
+            /* ... */
```

Comment:
This can go, as well as the others `/* ... */` on top of functions

Reply:
removed